RecycleBinThemes.ps1

```diff
@@ -3,9 +3,9 @@
 #
 
 $version = "v1.0.0"
-$pictures_path = [Environment]::GetFolderPath("MyPictures")
+#$pictures_path = [Environment]::GetFolderPath("MyPictures")
 # This is where we will store the icons
-$recycle_bin_themes_path = "$pictures_path\RecycleBinThemes"
+$recycle_bin_themes_path = "$env:userprofile\Pictures\RecycleBinThemes"
```

Comment:
Not sure how correct this is. What would happen if a user uses another language and does not have a directory called Picture? Or maybe they renamed it to "pics"

Reply:
I've checked it on a Dutch (my native language) W11 Virtual Machine I have running and even though I've opened the folder "Afbeeldingen" which is Dutch for pictures the file paths are in English. I don't know if that's for every language. ![image](https://user-images.githubusercontent.com/94373293/170362344-7472c43d-7a9a-4377-95a7-37cf187bf5ae.png)

Reply:
also, when I rename the folder to pics the path is still \Pictures